quizara-backend/index.js

```diff
@@ -1,7 +1,29 @@
-// import express from 'express';
+const express = require('express');
+var bodyParser = require('body-parser')
```

Comment:
No need to use `body-parser` you can use pre-built `express.json()`

Link:
http://expressjs.com/en/api.html